Makefile

```diff
@@ -12,6 +12,10 @@ NODE_MODULES_TEST=package-lock.json
 # one example file in dist/ from webpack to check if that already ran
 WEBPACK_TEST=dist/index.html
 
+WEBLATE_REPO=cockpit-podman-weblate
+WEBLATE_REPO_URL=git@github.com:cockpit-project/cockpit-podman-weblate.git
+WEBLATE_REPO_BRANCH=master
```

Comment:
This is a bit inconvenient to  hardcode, as you have to remember to change it when branching (this bit me several times from zanata.xml as well). Can we get away with just using `$(shell git rev-parse --abbrev-ref HEAD)` instead of using a variable? Or at least set the variable to that value?

Reply:
Yeah, I know we were switching this on branching time in zanata.xml, so I thought it is fine to keep it here and we can switch it.
I like the idea that we can parse it. I'll try it with the command you provided.

Reply:
Hmm, does it mean that I won't be able to do the refresh from this branch? Because bots prepare this branch and then do `po-refresh`, but this branch does not exist in weblate repo...

Reply:
As discussed on IRC, this doesn't work well with directly checking out refs from PRs, so let's keep the hardcoding for now indeed.